zulipterminal/config.py

```diff
@@ -1,18 +1,42 @@
 KEY_BINDINGS = {
```

Comment:
Should we rename this file to `keyconfig`?  Or do we expect to add other config to this file?

Reply:
It would probably be nice to have a developer tool (+ linter that runs on CI?) that can automatically generate the README section for keybindings from this data-structure.

Reply:
I do have other config I plan to add to this file.